src/enrich/lib/controllers/csv.js

```diff
@@ -111,20 +78,22 @@ const flatten = (obj) => {
   }
   flattenProp(obj, []);
   return flattened;
-};
+}
 
 /**
- * enrich file data with data from a ezunpaywall
- * @param {array<object>} data - array of line that we will enrich
- * @param {array<object>} response - response from ezunpaywall
- * @returns {array<object>} enriched data
+ * Enrich data with response from ezunpaywall
+ *
+ * @param {array<object>} data - Array of line that we will enrich.
+ * @param {array<object>} response - Response from ezunpaywall.
+ *
+ * @returns {array<object>} Number of line enriched and enriched data.
```

Comment:
The type is `Array` not `array`
The type is `Object` not `object`
```suggestion
 * @param {Array<Object>} data - Array of line that we will enrich.
 * @param {Array<Object>} response - Response from ezunpaywall.
 *
 * @returns {Array<Object>} Number of line enriched and enriched data.
```